examples/counter/src/lib.rs

```diff
@@ -105,5 +105,5 @@ fn view(model: &Model) -> impl View<Msg> {
 
 #[wasm_bindgen(start)]
 pub fn render() {
-    seed::App::build(|_, _| Init::new(Model::default()), update, view).build_and_start();
+    seed::App::builder(update, view).build_and_run();
```

Comment:
Why you changed `build_and_start` to `build_and_run`?

Reply:
I need to combine the two, actually. They took two different code paths that I didn't realize could be combined until right before I went to sleep.